application/services/studies/create_study.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'dry/transaction'
+require 'http'
+
+module SurveyMoonbear
+  module Service
+    # Returns a new study, or nil
+    # Usage: Service::CreateStudy.new.call(config: <config>, current_account: {...}, params: {...})
+    class CreateStudy
+      include Dry::Transaction
+      include Dry::Monads
+
+      step :store_into_database
+      step :get_study_arn
+      step :update_study_arn
+
+      private
+
+      def store_into_database(input)
+        track_activity = !input[:params]['track_activity'].nil?
+        enable_notification = !input[:params]['enable_notification'].nil?
+        input[:params].update('track_activity' => track_activity,
+                              'enable_notification' => enable_notification,
+                              'aws_arn' => 'checking enable notification or not')
+        new_study = Mapper::StudyMapper.new.load(input[:params], input[:current_account])
```

Comment:
Talk to mapper obj only when creating a study. Other cases(read, delete and update) did not talk to mapper obj.
Not sure if the timing is correct to use mapper like this.